src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -9156,5 +9156,54 @@ public void PreprocessingSymbol()
                 ],
                 actual: displayParts);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/36654")]
+        public void MinimalNameWithConflict()
+        {
+            var source =
+                """
+                using System;
+                namespace Abc.System {}
+                namespace Abc
+                {
+                    public class Test
+                    {
+                        public void fff()
+                        {
+                            Video vvv = null;
+                            foreach (var ooo in vvv.getlist()) {}
+                        }
+                    }
+                }
+
+                class Video
+                {
+                    public System.Collections.Generic.IEnumerable<int> getlist() { return null; }
+                }
+                """;
+
+            var compilation = CreateCompilation(source);
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+
+            var invocation = tree.GetRoot().DescendantNodes().OfType<InvocationExpressionSyntax>().Single();
```

Comment:
nit: FYI, we have a helper I like to use that lets you do `var invocation = GetSyntax<InvocationExpressionSyntax>(tree, "vvv.getlist()");`
The advantage is that it is immediately apparent to reviewers what invocation syntax we're grabbing